vue.config.js

```diff
@@ -0,0 +1,25 @@
+const { defineConfig } = require('@vue/cli-service')
+module.exports = defineConfig({
+  transpileDependencies: true,
+})
+
+module.exports = {
+  chainWebpack: config => {
+    if (config.plugins.has('extract-css')) {
```

Comment:
Unclear if we've got a SASS processor in the loop here...?

Reply:
I've confirmed that the exported app picks up and deals with Sass code properly.